enterprise/frontend/src/embedding-sdk/components/private/FlexibleSizeComponent.tsx

```diff
@@ -1,16 +1,27 @@
 import type { CSSProperties, PropsWithChildren } from "react";
 
+import type { CommonElementProps } from "embedding-sdk/types/props";
```

Comment:
nit: I'd name this more specific. Something like `CommonStylingProps` would make it super clear.

`CommonElementProps` could mean anything and it's broad. If we think we're gonna add more props to this type later on then `CommonElementProps` would make more sense, but for now it's just `CommonStylingProps`

Reply:
Agree.